src/levanter/compat/torch_serialization.py

```diff
@@ -55,9 +56,9 @@ def from_state_dict(self: Mod, state_dict: StateDict, prefix: Optional[str] = No
     def update_state_dict(self, state_dict: StateDict, prefix: Optional[str] = None) -> StateDict:
         return default_update_state_dict_with_eqx_module(state_dict, self, prefix)
 
-    def _state_dict_key_map(self) -> Optional[Dict[str, Optional[str]]]:
+    def _state_dict_key_map(self) -> Dict[str, Optional[str]]:
         """Returns a dict mapping eqx.Module keys to torch keys that need to be renamed for serialization"""
-        return None
+        return {}
```

Comment:
This change meant that I could do `_state_dict_key_map().get(...)` without special-casing.